pkg/kudoctl/cmd/test.go

```diff
@@ -150,6 +152,10 @@ For more detailed documentation, visit: https://kudo.dev/docs/testing`,
 				options.ArtifactsDir = artifactsDir
 			}
 
+			if isSet(flags, "report") {
+				options.ReportFormat = strings.ToLower(reportFormat)
```

Comment:
Will `testutils.RunTests` complain if this option isn't "json" or "xml"? I.e. do we need our own validation?

Reply:
yeah... we should put some validation in kuttl for this... I'll create an issue.

Reply:
https://github.com/kudobuilder/kuttl/issues/153

Reply:
to round out the conversation, there is NO invalid input... The default of "" is no report... but anything else defaults to json and "json" and "xml" are expected but lowercase.   We will fix some of this in the kuttl issue listed.